app/controllers/catchall_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CatchallController < ApplicationController
+  def not_implemented
+    interactor.call do |error|
+      if error.redirect?
+        return redirect_to error.location, status: error.status
+      end
+    end
+
+    raise ActionController::RoutingError.new('Not Found')
```

Comment:
this uses the current implementation found here https://github.com/moneyadviceservice/frontend/blob/master/app/controllers/application_controller.rb#L33 that rest of the site uses